ldms/src/ldmsd/ldmsd_request.c

```diff
@@ -613,7 +613,6 @@ int is_req_id_priority(enum ldmsd_request req_id)
 	case LDMSD_ENV_REQ:
 	case LDMSD_VERBOSE_REQ:
 	case LDMSD_LISTEN_REQ:
-	case LDMSD_AUTH_ADD_REQ:
```

Comment:
Why did you need to remove auth_add from the priority command list? Did it give you problems with ldmsd_communicator?

Reply:
Thanks for looking at this @nichamon! I removed auth_add from the priority command list because it seemed to be having the opposite effect as intended. The event I was experiencing was the ldmsd would attempt to add a producer to an aggregator before adding the authentication domain

Reply:
What was the error message? Did you configure LDMSD using a configuration file when you saw the problem? The request priority list applies to the configuration files specified with the -c cmd-line option.

Reply:
I believe it was just that the aggregator was attempting to add a producer before adding the authentication domain (i.e. auth domain didn't exist). I'm going to rebase this on top of a connection timeout PR I'll be making to maestro this week, so I'll test it again with AUTH_ADD in the priorities list at that time and give you more detailed info.

Reply:
@nick-enoent Any updates on the errors caused by the auth_add command in the priority list?

Reply:
@nichamon I'm no longer seeing an issue with keeping AUTH_ADD in the request priority list

Reply:
@nick-enoent Thanks for checking it! Could you please update the patch, so it doesn't remove the auth_add from the priority list?

Reply:
@nichamon I'm almost done with this patch. It has quite a few changes (along with keeping AUTH_ADD in the priority list. I'll make sure to ping you when I have pushed the new commit.

Reply:
@nick-enoent Thanks for the update.